pkgs/dart_tooling_mcp_server/lib/src/mixins/pub_dev_search.dart

```diff
@@ -24,6 +21,8 @@ final _resultsLimit = 10;
 
 /// Mix this in to any MCPServer to add support for doing searches on pub.dev.
 base mixin PubDevSupport on ToolsSupport {
+  final _client = Client();
```

Comment:
Not sure if this is a good idea but it's _an_ idea 🤣 :

You could make this late and then it won't get created until first use, so you could restore the `setUp` logic in the test so the style would get a bit closer to the other tests, only requiring one special zone function.